ui/src/pages/Dashboard.tsx

```diff
@@ -16,82 +16,83 @@ import {
 } from "@fortawesome/free-brands-svg-icons";
 import { faDownload } from "@fortawesome/free-solid-svg-icons";
 
-import CardImage from "../apex.png";
+// TODO: Replace the following with a Nexodus logo and uncomment the follow two blocks
+// import CardImage from "../apex.png";
```

Comment:
The new logo is in tree now

Reply:
@russellb Added this one `../../../assets/wordmark.png`

Forgot to lint, ty!